schema/schema.sql

```diff
@@ -4,7 +4,7 @@ CREATE TABLE music_album (
   author INTEGER,
   genre INTEGER,
   source INTEGER,
```

Comment:
kindly, remove the reference to source there is no such a table in this schema.

Reply:
Dear @amexabee ,

Kindly check the revised code and advise if its okay now.